7024Movies/DiscoverMovieSchema.json

```diff
@@ -35,10 +35,13 @@
       "additionalProperties": false,
       "properties": {
         "popularity": {
-          "type": "number"
+          "type": "number",
+          "minimum": 0,
```

Comment:
Based on the scope of the project, although it is sometimes unknown it is still best to give the minimum and maximum range for the popularity range. This help shelve out irregular values. 
https://www.w3.org/WAI/tutorials/forms/validation/

Reply:
Good idea.  Reasonable values never hurt.  :)